commit bfde03a14962c380d0e43327142710718c351cdb
Author: Martin Pitt <martin.pitt@ubuntu.com>
Date:   Mon Sep 7 19:37:43 2009 +0200

    modem-modeswitch rules: Match more devices
    
    extras/modem-modeswitch/61-mobile-action.rules: Match on device class/subclass
    "00" as well, some devices like the Vodafone K3565-Z have that.
    
    https://launchpad.net/bugs/281335

diff --git a/extras/modem-modeswitch/61-mobile-action.rules b/extras/modem-modeswitch/61-mobile-action.rules
index 9f47231..9116b25 100644
--- a/extras/modem-modeswitch/61-mobile-action.rules
+++ b/extras/modem-modeswitch/61-mobile-action.rules
@@ -1,6 +1,6 @@
 ACTION!="add", GOTO="mobile_action_end"
 
-SUBSYSTEM=="usb", ATTR{bDeviceClass}=="ff", ATTR{bDeviceSubClass}=="ff", ENV{DEVTYPE}=="usb_device", GOTO="mobile_action_switch"
+SUBSYSTEM=="usb", ATTR{bDeviceClass}=="00|ff", ATTR{bDeviceSubClass}=="00|ff", ENV{DEVTYPE}=="usb_device", GOTO="mobile_action_switch"
 GOTO="mobile_action_end"
 
 LABEL="mobile_action_switch"
